### ISSUE TITLE
Issues running clamp after homebrew update

### ISSUE THREAD

**Issue by dominicpedro** [jide/clamp#45]
Since homebew updated yesterday, I am now getting an auth error when creating a new clamp install

ERROR 1698 (28000): Access denied for user 'root'@'localhost'

I believe mariadb now prevents running as root. Not sure if anyone else is getting this issue.

Any help would be greatly appreciated

**Comment by rqelibari** [comment]
Hi @dominicpedro,

can you tell a bit more about your system? What version of macOS or MacOS X are you running clamp on?

**Comment by jide** [comment]
Yeah, it seems Mariadb has changed a few things about root user. It seems it could work with passing `-u root` to the mysqld command, but another error is thrown: `unknown variable 'defaults-file=/dev/null'`.

As a workaround, you can install and use an older version of mariadb, and edit the clamp config file after having erased the `.clamp` folder : 
```
rm -rf .clamp
brew install mariadb@10.0
clamp config write
```
And replace `$(brew --prefix mariadb)` with `$(brew --prefix mariadb@10.0)` in `clamp.json`.

**Comment by dominicpedro** [comment]
Thanks for the update. Will try your workaround

Dom
